test/helpers/kubectl.go

```diff
@@ -548,6 +548,18 @@ func (kub *Kubectl) GetCiliumPodOnNode(namespace string, node string) (string, e
 	return res.Output().String(), nil
 }
 
+// TestConnectivityPodService runs HTTP connectivity test from pod to ClusterIP service
+func (kub *Kubectl) TestConnectivityPodService(from, to string) *CmdRes {
+	kub.logCxt.WithFields(log.Fields{
```

Comment:
Nit: would be nice if instead of a string for IPs we could pass in actual IP objects. This can be done in a follow-up ticket. We should utilize golang constructs in the tests when we can instead of strings.

Reply:
None of arguments passed to this method is ip, `from` is a pod name, `to` is a service name.